app/src/main/java/org/wikipedia/page/PageActivity.kt

```diff
@@ -568,7 +568,9 @@ class PageActivity : BaseActivity(), PageFragment.Callback, LinkPreviewDialog.Ca
             wasTransitionShown = true
         }
         if (entry.source != HistoryEntry.SOURCE_INTERNAL_LINK || !Prefs.isLinkPreviewEnabled) {
-            LinkPreviewFunnel(app, entry.source).logNavigate()
+            val articleLinkPreviewInteractionEvent = ArticleLinkPreviewInteractionEvent(pageTitle.wikiSite.dbName(),
+                pageFragment.page?.pageProperties?.pageId!!.or(0), entry.source)
```

Comment:
Incorrect usage of `or()`. The `or()` function is really just the bitwise OR operator, unrelated to null conditionals.
The intention here is probably to use the elvis operator `?:`.